src/terms.jl

```diff
@@ -483,6 +483,13 @@ modelcols(t::Term, d::NamedTuple) =
 modelcols(ft::FunctionTerm{Fo,Fa,Names}, d::NamedTuple) where {Fo,Fa,Names} =
     ft.fanon.(getfield.(Ref(d), Names)...)
 
+# this is weird, but using import Base: copy leads to exporting type piracy
+# for non missing values, the compiler should hopefully optimize down the extra
+# layer of indirection
+function copy end
+copy(x::Any) = Base.copy(x)
+copy(m::Missing) = m
```

Comment:
I think this feels like a bad idea.  Maybe better to just drop the use of `copy` (and broadcast `identity` or use `copy` on the vector).